.github/CODEOWNERS

```diff
@@ -6,4 +6,4 @@
 cli/*   @shaunmitchellve
 
 # Solution Reviewers
-solutions/* @davelanglois-ssc @alaincormier-ssc
\ No newline at end of file
+solutions/* @davelanglois-ssc @alaincormier-ssc @lucstjean-ssc @Michael-McAuley-SSC @amcmullin @CraigBaltzer @uday-ssc @ovidiuasiminei-ssc @stephanemillaire-ssc
```

Comment:
@davelanglois-ssc @alaincormier-ssc @lucstjean-ssc @Michael-McAuley-SSC @amcmullin @CraigBaltzer @uday-ssc @ovidiuasiminei-ssc @stephanemillaire-ssc @cheffj @richtremblay-ssc @adamahmed6 @danielsvensson-ssc @seanwilkens-ssc

Reply:
Thanks for the list @davelanglois-ssc . I must have been looking at an older email.

Reply:
do you think seeting all these persons as code owners will make each one of them required reviewers

Reply:
I think it would pull from the pool of owners for reviewers. Currently the required reviewers are set to 2 but we can up that. I will also check to see if we can set a list of required reviewers.